boards/risc-v/bl602/bl602evb/src/bl602_bringup.c

```diff
@@ -149,6 +151,23 @@ int bl602_bringup(void)
     }
 #endif
 
+#ifdef CONFIG_DEV_GPIO
+  ret = bl602_gpio_initialize();
+  if (ret < 0)
+    {
+      syslog(LOG_ERR, "Failed to initialize GPIO Driver: %d\n", ret);
+      return ret;
+    }
+#endif
+
+#ifdef CONFIG_I2C 
+  ret = bl602_i2cbus_initialize(0);
```

Comment:
Can you also hook in the i2ctool support here (you can see it in my PR).  Makes it easier to do testing.